mu4/notation/inotation.h

```diff
@@ -45,6 +45,8 @@ class INotation
     virtual Meta metaInfo() const = 0;
 
     virtual void setViewSize(const QSizeF& vs) = 0;
+    virtual void setViewMode(const ViewMode& vm) = 0;
+    virtual ViewMode getViewMode() = 0;
```

Comment:
Usually, we ignore the "get" word in the getters and they are usually "const" methods